tests/unit/lms/views/lti/oidc_test.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+from h_matchers import Any
+from pyramid.httpexceptions import HTTPForbidden, HTTPFound
+
+from lms.views.lti.oidc import oidc_view
+
+
+class TestOIDC:
+    def test_missing_registration(self, lti_registration, pyramid_request):
+        lti_registration.get.return_value = None
+
+        with pytest.raises(HTTPForbidden):
+            oidc_view(pyramid_request)
+
+        lti_registration.get.assert_called_once_with(
+            issuer=pyramid_request.parsed_params["iss"],
+            client_id=pyramid_request.parsed_params["client_id"],
+        )
```

Comment:
We test the same thing in both tests, but in different ways. I'd probably proved this the once in one place and leave it in the other.